subsys/bluetooth/shell/bt.c

```diff
@@ -233,6 +303,13 @@ static void scan_recv(const struct bt_le_scan_recv_info *info,
 		    phy2str(info->primary_phy), phy2str(info->secondary_phy),
 		    info->interval, BT_CONN_INTERVAL_TO_US(info->interval),
 		    info->sid);
+	if (scan_verbose_output) {
+		shell_info(ctx_shell,
+				   "          [SCAN DATA START - %s]",
+				   scan_response_type_txt[info->adv_type]);
+		bt_data_parse(&buf_copy, data_verbose_cb, NULL);
+		shell_info(ctx_shell, "          [SCAN DATA END]");
```

Comment:
Ditto with using `\t` instead of fixed number of spaces. 

In the case that you want to keep the spaces, I suggest to define the spacing somewhere if we want to use the same spacing multiple places.